.env.sample

```
@@ -21,6 +20,11 @@ DATAGOUV_API_KEY=
 
 API_DEPOT_URL=https://plateforme.adresse.data.gouv.fr/api-depot
 
+API_IDEFIX_URL=https://plateforme.adresse.data.gouv.fr/api-idefix
+API_IDEFIX_TOKEN=
```

```suggestion
API_IDFIX_URL=https://plateforme.adresse.data.gouv.fr/api-idfix
API_IDFIX_TOKEN=
```

Je sais que tu aimes le nom `Idéfix` mais la brique s'appelle `id-fix` 😄